api/src/main/java/org/pmiops/workbench/db/model/Workspace.java

```diff
@@ -152,11 +151,11 @@ public void setFirecloudName(String firecloudName) {
   }
 
   @Column(name = "data_access_level")
-  public DataAccessLevel getDataAccessLevel() {
+  public Short getDataAccessLevel() {
```

Comment:
I think it'd be nice if whenever using Short like this, we added a comment indicating the enum class it corresponds to / what method to use to convert.

We *could* even add getters / setters to the model objects using the enum classes (which JPA would ignore)... the downside being we'd still have a dependency on Swagger in our model objects (which makes our build and particularly tools build messier.)

Reply:
If it's tricky, don't bother with the extra methods; but let's document what enum value we're using here.

Reply:
Several hours later, turns out my suspicions were also confirmed and Spring data dark magic of course got in the way. Apparently it will attempt to glean some meaning from public method names on your entity. I'm not sure of the exact rules or why it is doing this when it already has explicit annotations, but this code appears to be involved somehow: https://github.com/hibernate/hibernate-orm/blob/736c913f1428e16a5a3ef26b52d9ec91ff323f25/hibernate-core/src/main/java/org/hibernate/type/TypeResolver.java#L95

At least one trigger for this seems to be having a getter/setter-like syntax. So unfortunately these helpers will need to have slightly unnatural naming, I went with an "enum" prefix which dodges the problem.

I'm almost tempted to revert this because I really dislike being at the mercy of Spring's opaque automagical model handling, but I think it's probably worth going with this version since the client code is so much cleaner.